.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+/kind bug
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+
+**What did you expect to happen:**
+
```

Comment:
Would you mind adding "How to reproduce it (as minimally and precisely as possible):" to the bug report?

Reply:
@gaoning777 it was already covered in `**What steps did you take and what happened:**` in Line 9